junit.xml

```diff
@@ -0,0 +1,538 @@
+<?xml version="1.0" encoding="utf-8"?><testsuite errors="0" failures="0" name="pytest" skips="6" tests="6" time="2.060"><testcase classname="tests.test_win_vs_activate" file="tests\test_win_vs_activate.py" line="73" name="test_activation_logic[32-vs14.0]" time="0.11809563636779785"><skipped message="expected test failure">verification of test logic</skipped><system-out>writing C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\vcvarsall.bat (exit /b 1)
+writing c:\Program Files (x86)\Microsoft Visual Studio 10.0\VC\vcvarsall.bat (exit /b 1)
+writing C:\Program Files\Microsoft SDKs\Windows\v7.1\Bin\SetEnv.cmd (exit /b 1)
+writing C:\Users\pts08\AppData\Local\Programs\Common\Microsoft\Visual C++ for Python\9.0\vcvarsall.bat (exit /b 1)
+writing C:\Program Files (x86)\Microsoft Visual Studio 9.0\VC\bin\vcvars64.bat (exit /b 1)
+

+C#x08L#x08I#x08N#x08K#x08 #x08#x1B[1;32;40mC:\Users\pts08\Documents\GitHub\conda-build {git}{hg} 

+#x1B[1;30;40m{lamb} #x1B[0mset DISTUTILS_USE_SDK=1 

+

+C#x08L#x08I#x08N#x08K#x08 #x08#x1B[1;32;40mC:\Users\pts08\Documents\GitHub\conda-build {git}{hg} 


```

Comment:
Should this file be in the commit? The output is from a cmder session (`{git}{hg}` and `{lamb}`) and won't be the same in a plain cmd